libraries/integration/Microsoft.Bot.Builder.Integration.Runtime/Extensions/ConfigurationBuilderExtensions.cs

```diff
@@ -0,0 +1,177 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.Extensions.Configuration;
+
+namespace Microsoft.Bot.Builder.Integration.Runtime.Extensions
+{
+    /// <summary>
+    /// Extensions for setting up Runtime IConfiguration.
+    /// </summary>
+    public static class ConfigurationBuilderExtensions
+    {
+        private const string AppSettingsFileName = @"appsettings.json";
```

Comment:
We should also support appsettings.Development.json (which is standard for aspnet apps and devs use to override default settings), maybe I am messing something but we don't seem to be supporting it.